src/main/java/com/zendesk/maxwell/producer/HttpPostProducer.java

```diff
@@ -0,0 +1,115 @@
+package com.zendesk.maxwell.producer;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpResponseException;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.HttpHeaders;
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpBackOffUnsuccessfulResponseHandler;
+import com.google.api.client.util.ExponentialBackOff;
+
+import com.google.api.client.http.javanet.NetHttpTransport;
+import org.apache.commons.codec.binary.Hex;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import java.io.IOException;
+
+public class HttpPostProducer extends AbstractProducer {
+    static final String HTTP_HMAC_HEADER = "x-signature";
+    static final String HTTP_CONENT_TYPE_HEADER = "application/json";
+    static final String HMAC_ALG = "HmacSHA1";
```

Comment:
how about following spec [here](https://tools.ietf.org/html/draft-cavage-http-signatures-05) and providing HMAC via authorization, e.g., `Authorization: Signature keyId="hmac-key-1",algorithm="hmac-sha1",signature="Base64(HMAC-SHA1(signing string))"`. note optional headers are excluded for body signature verification mode. Also, what if we had `HmacAuthHttpPostProducer` as an example of an `AbstractHttpPostProducer` where users can implement their own headers, etc.